infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/Constants.java

```diff
@@ -50,5 +50,17 @@
   public static final String POD_STATUS_PHASE_FAILED = "Failed";
   public static final String POD_STATUS_PHASE_SUCCEEDED = "Succeeded";
 
+  /** The projects volume has a standard name used in a couple of locations. */
+  public static final String PROJECTS_VOLUME_NAME = "projects";
+
+  /** The mount path of the projects volume in the tooling containers */
+  public static final String PROJECTS_MOUNT_PATH = "/projects";
+
+  /**
+   * The name of the environment variable that contains injected into the containers that mount
+   * sources
+   */
+  public static final String PROJECTS_MOUNT_PATH_ENV_VAR_NAME = "CHE_SOURCE_ROOT";
```

Comment:
There is a similar ENV_VAR https://github.com/eclipse/che/blob/07263f1e30089689d71b057f747a44a29283e3c4/wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/spi/provision/env/ProjectsRootEnvVariableProvider.java#L27
For some reasons, I don't see where it is bound, but it should be. 
Please avoid duplication if possible.